[PROOFREADING] SCU101 - German

maggo83 opened the pull request:
"Sie" -> "du" (more personal)
more natural formulations

closes #2547

maggo83 committed:
Update de.md - upto chapter 3 (Computer use)

Proofread chapters 1 and 2
change from "Sie" to "Du".
More natural phrasing

maggo83 committed:
Updated de.md: everything until "Practical" section

Formal -> personal
natural language

maggo83 committed:
done prrofreading de.md

maggo83 committed:
Update de.md

maggo83 commented:
Ideas for improving the original material:
* in the https section: 
** "Therefore, it is essential to verify that the site you are visiting uses the HTTPS protocol." -> maybe add: "by checking the address line in your browser" or similar (in the intro it is claimed, the training is also for non technical readers).
** "It is important to verify the site's public key using a certificate system to confirm its authenticity." -> How? Non technical readers will not know
* add VPN to glossary?
* add 2FA to glossary?
* "One-time passwords (HOTP and TOTP) are safer than SMS because they require cryptographic calculation and are stored locally rather than in memory." -> what does "rather than in memory" mean? why is that unsafe?

maggo83 committed:
Update de.md

Found two paragraphs in the english original that were completely not present in the auto-tranlated version. translated them manually

MariJJhodl commented:
Hey @maggo83 nice suggestions, thank you. If you have any practical suggestion on what to add in the https section (certificate system) and One-time passwords (HOTP and TOTP) part, feel free to write them: we will check and eventually add them to all languages!

As for VPN and 2FA, I can suggest to add them to the glossary or we can find an extra link for beginners, to add to all languages.

maggo83 committed:
Added proposals for https / HOTP/TOTP section

Added some small changes to make details a bit more clear. Please double check for factual correctness, I am no securtiy expert myself and basically googled this

MariJJhodl commented:
Hey @LoicPandul, a couple of sentences were added to this course, both in German and English. Can you check the EN version and give me a second opinion? Should we approve the proposals?

LoicPandul reviewed (approved):
Ok for en.md, thanks ! :)

maggo83 commented:
While this is still open, another question/proposal:

@LoicPandul :
In the password manager section, self-hosted PMa are mentioned. Wouldn't it make sense to also introduce some good solutions for synching the password files between different devices, like e.g. syncthing in combination with keepass?

LoicPandul committed:
Merge remote-tracking branch 'upstream/dev' into pr/2646

LoicPandul committed:
add paragraph PM sync

LoicPandul commented:
Nice idea @maggo83 thanks! I just added a paragraph to explain this on your branch in all languages + fixed the conflicts with dev
